docs/get-started.md

```diff
@@ -28,8 +28,20 @@ docker pull vesoft/nebula-graph:latest
 to get `nebula` image and `docker images` that can display images status.
 
 If `docker` is slow when pulling the image, configure an accelerator.
+1. Check if the `/etc/docker` folder exists, if not create a new one with `mkdir -p /etc/docker`. Generally the folder will exist after docker installation.
```

Comment:
Is there a comma needed after _if not_? And a _the_ before `docker installation`?

Reply:
Yes, a comma is better but I don't think ' a the before `docker installation`' is needed.